pkgs/by-name/se/secretlint/package.nix

```
@@ -0,0 +1,70 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  nix-update-script,
+  makeWrapper,
+  nodejs,
+  pnpm_10,
+  versionCheckHook,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "secretlint";
+  version = "11.0.2";
+
+  src = fetchFromGitHub {
+    owner = "secretlint";
+    repo = "secretlint";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-LJ1zYv0w7e9fPLfhzfCdasloRUdcy8lfMIQTyHqAY/Y=";
+  };
+
+  pnpmDeps = pnpm_10.fetchDeps {
+    inherit (finalAttrs) pname version src;
+    fetcherVersion = 2;
+    hash = "sha256-jEmpyj83ORKUWHzVYoFzKdaB7epLam33nP7t0z5QmlE=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+    nodejs
+    pnpm_10.configHook
+  ];
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgramArg = "--version";
+  doInstallCheck = true;
```

```suggestion
  doInstallCheck = true;
  nativeInstallCheckInputs = [ versionCheckHook ];
```

`--version` should be the default.